tests/BrowsershotTest.php

```diff
@@ -162,20 +162,6 @@ public function it_can_handle_a_permissions_error()
             ->save($targetPath);
     }
 
-    /** @test */
-    public function it_can_use_the_methods_of_the_image_package()
```

Comment:
I removed this test because there's method named `format()` is overrided. So we cannot access to spatie/image package's function.
(I guess `format()` function is implemented for setting pdf format)